util/chplenv/utils.py

```diff
@@ -62,3 +62,20 @@ def run_command(command, stdout=True, stderr=False, cmd_input=None):
         else:
             return ''
 
+def run_live_command(command):
+    """Run a command, writing its output live instead of after the process is
```

Comment:
Will adjust the docs.


I was originally trying to make `live` an argument to run_command, but I gave up on that because the stdin stuff got weird and because it doesn't really make sense to ignore stdout/stderr for the live version. i.e. in order to support stdin with the live version, you end up having to do `stdin.write(input)`, which isn't recommended. communicate() is recommended to avoid deadlock due to OS pipe buffers filling up, but you can't do live output with communicate so I ended up just creating a second function that doesn't accept intput. There's also no reason to use the live version if you're going to ignore the output so I also removed the stdout and stderr arguments